zendesk/app_test.go

```diff
@@ -32,7 +32,7 @@ func TestListAppInstallations(t *testing.T) {
 			},
 			SettingsObjects: []struct {
 				Name  string `json:"name"`
-				Value string `json:"value"`
+				Value any    `json:"value"`
```

Comment:
this was a failing unit test i had committed to master from a previous patch, lol oops